__tests__/app.test.js

```diff
@@ -126,8 +126,8 @@ describe.only("/api/articles", () => {
   });
 });
 
-describe("/api/articles/:article_id", () => {
-  test("should get an article object specified by the endpoint", () => {
+describe("GET/api/articles/:article_id", () => {
+  test("should get an article object (with comment count) specified by the endpoint", () => {
     return request(app)
```

Comment:
We don't want to change any existing tests - as this is not in the spirit of TDD! Instead, we want to make a new test that is now checking for comment_count as well, and if the previous test was good and not too brittle, it will work